src/react/ui-elements/Veeam/VeeamConfiguration.tsx

```diff
@@ -0,0 +1,205 @@
+import { Button, Input, Select } from '@scality/core-ui/dist/next';
+import { useRef } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  version: Joi.string().required(),
+  capacity: Joi.number().required().min(1).max(1023),
+  capacityUnit: Joi.string().required(),
+  enableImmutableBackup: Joi.boolean().required(),
+});
+
+type VeeamConfiguration = {
+  name: string;
+  version: string;
+  capacity: string;
+  capacityUnit: string;
+  enableImmutableBackup: boolean;
+};
+
+const Configuration = () => {
+  const {
+    handleSubmit,
+    control,
+    register,
+    formState: { errors, isValid },
+  } = useForm<VeeamConfiguration>({
+    mode: 'all',
+    resolver: joiResolver(schema),
+    defaultValues: {
+      name: '',
+      version: 'Veeam12',
+      capacity: '5', //TODO: The default value will be net capacity.
+      capacityUnit: 'TiB',
+      enableImmutableBackup: true,
+    },
+  });
+
+  const onSubmit = () => {
+    //TODO: Create account
+  };
+  const formRef = useRef(null);
+  return (
+    <Form
+      onSubmit={handleSubmit(onSubmit)}
+      ref={formRef}
+      layout={{
+        title: 'Configure ARTESCA for your Use case',
+        kind: 'page',
+      }}
+      rightActions={
+        <Stack gap="r16">
+          <Button
+            type="button"
+            variant="outline"
+            onClick={() => {
+              //TODO: Go back to the landing page base on the user profile
+            }}
+            label="Skip Use case configuration"
+          />
+          <Button
+            type="submit"
+            variant="primary"
+            label="Continue"
+            disabled={!isValid}
+          />
+        </Stack>
+      }
+    >
+      <FormSection title={{ name: 'Prepare ARTESCA for Veeam' }}>
+        <FormGroup
+          id="version"
+          label="Veeam version"
+          direction="vertical"
+          labelHelpTooltip="TODO"
+          content={
+            <Controller
+              name="version"
+              control={control}
+              render={({ field: { onChange, value } }) => {
+                return (
+                  <Select
+                    id="version"
+                    onChange={(value) => {
+                      onChange(value);
+                    }}
+                    value={value}
+                  >
+                    <Select.Option key="Veeam11" value={'Veeam11'}>
+                      Veeam 11
+                    </Select.Option>
+                    <Select.Option key="Veeam12" value={'Veeam12'}>
+                      Veeam 12
+                    </Select.Option>
+                  </Select>
+                );
+              }}
+            ></Controller>
+          }
+        ></FormGroup>
+        <FormGroup
+          id="name"
+          label="Repository name"
+          direction="vertical"
+          required={true}
```

Comment:
nit: this can be simplified 😃 
```suggestion
          required
```